proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -341,11 +336,13 @@ Though the property wrapper is initialized at the call-site, the argument type _
 ```swift
 @propertyWrapper
 struct Wrapper<Value> {
-
-  init(wrappedValue: @autoclosure () -> Value) { ... }
-
-  init(wrappedValue: @autoclosure () -> Value) where Value : Collection { ... }
+  init(
+    wrappedValue: @autoclosure () -> Value
+  ) { ... }
   
+  init(
+    wrappedValue: @autoclosure () -> Value
+  ) where Value : Collection { ... }
```

Comment:
I really prefer the old formatting where the function signature is on one line. I was writing the code that way deliberately. For this example in particular, it's a lot easier to spot the difference between the two initializers with the old formatting.

Reply:
Oh, I see. I still think it's hard to parse this line on small screen. Do you think that the following may read better?

```swift
init(wrappedValue: @autoclosure () -> Value)

init(wrappedValue: @autoclosure () -> Value)
  where Value : Collection
```

Reply:
I took `@autoclosure` out of this example so the line is much shorter now